libdebug/libdebug.py

```diff
@@ -1373,15 +1414,15 @@ def debugger(
 
     debugging_context = create_context(debugger)
 
-    debugging_context.clear()
```

Comment:
Was there anything wrong with clearing the debugging context? It should set up some attributes

Reply:
Some attributes are set when the context is created, and others are set after its creation. Clearing at startup duplicates some operations and does not allow for delayed initialization.